packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -261,6 +261,9 @@ export default class Meetings extends WebexPlugin {
     const isSelfMoved = newLocus?.self?.state === _LEFT_ && newLocus?.self?.reason === _MOVED_;
     // @ts-ignore
     const deviceFromNewLocus = MeetingsUtil.getThisDevice(newLocus, this.webex.internal.device.url);
+    const isResourceMovedOnThisDevice =
```

Comment:
Next time let's try handling the locus chang logic in the locus info file . The web client and the sdk team handle all the logic there .

I can have a session to explain how it works 

Meetings file should only handle meeting collection and routing events to the actual meeting object which then does further processing

Reply:
@arun3528 Yes, But for this case we need to check `isNeedHandleLocus` at the beginning of receiving locus DTO to cover the meeting object haven't create case. For example: user use native client start a meeting and join into bo, then user open web client with same account, web client may receive locusDTOs of Main and Breakout at same time, if not check the `isNeedHandleLocus` in meetings/index, may will create two meeting objects for these DTOs, will caused wrong behavior and issues. So I need to put the logic at the beginning position.

Reply:
got it thanks or the explanation